internal/cmd/commands/targetscmd/funcs.go

```diff
@@ -285,6 +323,15 @@ func extraFlagsFuncImpl(c *Command, _ *base.FlagSets, f *base.FlagSet) {
 			Usage:      "Target scope name, if authorizing the session via scope parameters and target name. Mutually exclusive with -scope-id.",
 		})
 	}
+
+	if c.Func == "set-static-address" {
+		flagsMap[c.Func] = append(flagsMap[c.Func], "address")
+		f.StringVar(&base.StringVar{
+			Name:   "address",
+			Target: &c.flagAddress,
+			Usage:  "Optionally, a valid network address to connect to for this target. Can not be used alongside host sources.",
```

Comment:
I believe this should be `cannot`.
```suggestion
			Usage:  "Optionally, a valid network address to connect to for this target. Cannot be used alongside host sources.",
```